tests/installation/welcome.pm

```diff
@@ -98,10 +98,9 @@ sub run {
             my %hotkey = (
                 sles     => 's',
                 sled     => 'u',
-                sles4sap => 'i',
-                hpc      => 'x'
+                sles4sap => get_var('OFW') ? 'u' : 'i',
```

Comment:
@ldevulder is it ok ? I haven't change a flow . just refactor a bit.

Reply:
Sounds good to me, but sles4sap tests have been done by @alvarocarvajald, so I let him the final word ;-)

Reply:
LGTM